AWS/rollapp.env

```diff
@@ -16,10 +16,10 @@ EVM_ENABLED=true
 # # Assuming 100,000,000REVM tokens
 # # evm uses 10^18 decimal precision for arevm
 #DENOM=arevm
-#result=$(echo "100 * 10^6 * 10^18" | bc)
-#staking_result=$(echo "$result / 2000" | bc)
-#TOKEN_AMOUNT="$result""$DENOM"
-#STAKING_AMOUNT="$staking_result""$DENOM"
+# result=100000000000000000000000000
```

Comment:
Why did you hardcode it?

Reply:
It cannot be generated within a .env file while the service is using it as an EnvironmentFile